Library/Formula/node.rb

```diff
@@ -5,6 +5,8 @@ class Node < Formula
   sha256 "0043656bb1724cb09dbdc960a2fd6ee37d3badb2f9c75562b2d11235daa40a03"
   revision 2
 
+  conflicts_with "iojs", :because => "io.js includes a symlink named node for compatibility."
```

Comment:
We don't need to add this. Homebrew will handle `conflicts_with` for both formulae. So the `conflicts_with` in iojs.rb is enough.